#goal
string cart_name
---
#result
bool cart_picked
string action_status
---